qa-tests-backend/scripts/run-compatibility.sh

```diff
@@ -63,15 +79,19 @@ compatibility_test() {
 
     update_junit_prefix_with_central_and_sensor_version
 
-    store_qa_test_results "compatibility-test-central-v${CENTRAL_CHART_VERSION_OVERRIDE}-sensor-v${SENSOR_CHART_VERSION_OVERRIDE}"
-    [[ ! -f FAIL ]] || die "compatibility-test-central-v${CENTRAL_CHART_VERSION_OVERRIDE}-sensor-v${SENSOR_CHART_VERSION_OVERRIDE}"
+    short_central_tag=$(shorten_tag "${CENTRAL_CHART_VERSION_OVERRIDE}")
```

Comment:
```suggestion
    short_central_tag="$(shorten_tag "${CENTRAL_CHART_VERSION_OVERRIDE}")"
```

Reply:
Oh, I thought you did not need quotation marks when evaluation inline commands. After some googling it seems that you are correct.

Reply:
I got two simple rules (that are not always necessary, but following them always makes it easy to remember):
1. Always quote exec `var="$(cmd)"`
2. Always quote vars `cmd sth sth "$var"`